src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,38 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+interface TodoModalProps {
+  chosenTodo: Todo,
+  nonExistedTodo: Todo,
+  setChosenTodo: (todo: Todo) => void,
+}
+
+export const TodoModal: React.FC<TodoModalProps> = ({
+  chosenTodo,
+  nonExistedTodo,
+  setChosenTodo,
+}) => {
+  const [user, setUser] = useState<User>();
+  const [showTodoModal, setShowTodoModal] = useState(true);
```

Comment:
[Tips on naming boolean variables](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isShowTodoModal, setIsShowTodoModal] = useState(true);
```